sql/src/main/java/io/crate/operation/user/User.java

```diff
@@ -57,6 +58,17 @@ public String name() {
         return privileges;
     }
 
+    public boolean isSuperUser() {
+        return roles.contains(Role.SUPERUSER);
+    }
+
+    public boolean hasPrivilege(Privilege.Type type, Privilege.Clazz clazz, String ident) {
```

Comment:
would be great to add docs for that method and mention that currently only cluster privileges can be created so checking just the type is enough